bpf/flows.c

```diff
@@ -13,11 +13,11 @@
             until an entry is available.
         4) When hash collision is detected, we send the new entry to userpace via ringbuffer.
 */
+#define BPF_NO_PRESERVE_ACCESS_INDEX
```

Comment:
can you add a comment explaining briefly the reason (or just linking to the JIRA and/or libpbf issue)  ?